sagemaker-featurestore/feature_store_object_detection_ground_truth.ipynb

```diff
@@ -0,0 +1,484 @@
+    "                        class_name_meta_data.append(d[k][j])\n",
+    "                    elif i2 == 1:\n",
+    "                        confidence_meta_data.append(d[k][j])\n",
+    "                    elif i2 == 2: \n",
+    "                        type_meta_data.append(d[k][j])\n",
+    "                    elif i2 == 3: \n",
+    "                        job_name_meta_data.append(d[k][j])\n",
+    "                    elif i2 == 4:\n",
+    "                        human_annotated_meta_data.append(d[k][j])\n",
+    "                    elif i2 == 5: \n",
+    "                        creation_date.append(d[k][j])\n",
+    "    return pd.DataFrame({\"item_name\" : item_name, \"classification\": classification, \"class_name_meta_data\": class_name_meta_data, \\\n",
+    "                        \"confidence_meta_data\": confidence_meta_data, \"type_meta_data\": type_meta_data, \\\n",
+    "                         \"job_name_meta_data\": job_name_meta_data, \"human_annotated_meta_data\": human_annotated_meta_data,\\\n",
+    "                        \"creation_date\": creation_date})\n",
+    "\n",
+    "df = create_dataframe_from_manifest('output.manifest')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Preview the parsed manifest file as a dataframe"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df.dtypes"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def cast_object_to_string(data_frame):\n",
+    "    \"\"\"\n",
+    "    data_frame: A pandas Dataframe \n",
+    "    \n",
+    "    Cast all columns of data_frame of type object to type string.\n",
+    "    \n",
+    "    Return updated data frame\n",
+    "    \"\"\"\n",
+    "    for label in data_frame.columns:\n",
+    "        if data_frame.dtypes[label] == object:\n",
+    "            data_frame[label] = data_frame[label].astype(\"str\").astype(\"string\")\n",
+    "    return data_frame"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Cast columns of df of type object to string. \n",
+    "df = cast_object_to_string(df)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create Feature Group and Ingest data into it\n",
+    "\n",
+    "Below we start by appending the `EventTime` feature to your data to timestamp entries, then we load the feature definition, and instantiate the Feature Group object. Then lastly we ingest the data into your feature store."
```

Comment:
"The following example starts by appending the `EventTime` feature to your data to timestamp entries, then loads the feature definition and instantiates the feature group object. Finally, it ingests the data into your feature store."